src/main/java/uk/gov/homeoffice/digital/sas/timecard/listeners/TimeEntryListener.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.homeoffice.digital.sas.timecard.listeners;
+
+import javax.persistence.PostPersist;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.homeoffice.digital.sas.timecard.model.TimeEntry;
+import uk.gov.homeoffice.digital.sas.timecard.producers.KafkaProducerTimeEntry;
+
+@Slf4j
+@Component
+public class TimeEntryListener {
+
+  private KafkaProducerTimeEntry kafkaProducerService;
+
+  @Autowired
+  public void createProducerService(KafkaProducerTimeEntry kafkaProducerService) {
+    this.kafkaProducerService = kafkaProducerService;
+  }
+
+  @PostPersist
+  private void prePersistEvent(TimeEntry entry) {
+    log.info("Running postPersistEvent");
```

Comment:
Not sure which way round is intended 😄 
```suggestion
    log.info("Running prePersistEvent");
```

Reply:
It is meant to be post so we don't interfere with anything within the dev environment. Data will still flow.